routing/notifications.go

```diff
@@ -318,11 +319,19 @@ func addToTopologyChange(graph *channeldb.ChannelGraph, update *TopologyChange,
 		if err != nil {
 			return err
 		}
+
+		globalFeaturesBuffer := new(bytes.Buffer)
+		err = m.Features.Encode(globalFeaturesBuffer)
+		if err != nil {
+			return err
+		}
+
 		nodeUpdate := &NetworkNodeUpdate{
-			Addresses:   m.Addresses,
-			IdentityKey: pubKey,
-			Alias:       m.Alias,
-			Color:       EncodeHexColor(m.Color),
+			Addresses:      m.Addresses,
+			IdentityKey:    pubKey,
+			GlobalFeatures: globalFeaturesBuffer.Bytes(),
```

Comment:
Golang question that's probably beyond this scope of this PR, but is there no way to require fields at compile time?

Reply:
Good question! It'd be nice but afaik unused fields are not reported by the compiler because it's normal to have fields initialized with default values in Go.